### ISSUE TITLE
[1.12] [Suggestion] More 3x3 Big Door Options

### ISSUE THREAD

**Issue by MokahTGS** [Ordinastie/MalisisDoors#173]
I would love if there were more big door options for all of the doors.  The big carriage door is 4x5 making it pretty awkward to use in most odd numbered builds which is my preferred way of building.

The best solution would be to add a 3x3 option for the door factory, but I'd settle for a 3x3 selection of doors.

Also the 3x2 high tech door doesn't work with the player sensor as far as I can tell.

**Comment by Ordinastie** [comment]
Easier said than done. Code wise, it's easy to do, but it still need models, textures, and eventually sounds. That's really not my forte.

**Comment by winsrp** [comment]
how about take the exiting textures and sound for normal doors and just make them bigger for a 3x3 for a start? there is a big doors mods that does that, and it is based on your mod, and it looks just fine, but is no longer updated.

**Comment by SgtSnipey** [comment]
I came here to ask about the Hi-Tech door for a 3x3 verizon, But more 3x3 would be cool as well :D
Great Mod Amazing job!

**Comment by ghost** [comment]
Can I suggest adding a big door option to the door factory? Even if you just stretched the textures 50% it would still be great.

**Comment by mcdodge34** [comment]
I don't like to wake-up these kind of old threads, but I need 3x3 doors from Malisis door factory that would open and detect the player, I want a massive entrance and not with the standard wooden doors, they don't slide or do any custom sounds like I want.

**Comment by ghost** [comment]
@mcdodge34 You might want to look at Little Tiles (https://minecraft.curseforge.com/projects/littletiles). It's like Chisels and Bits, but it has support for building doors, ladders, storage blocks, chairs...
There's an outdated video on the page which goes over the basics of each.

**Comment by MokahTGS** [comment]
But that doesnt give the options of the door factory.

**Comment by ghost** [comment]
True. There are ways to get a custom sound by using redstone to other mods, but AFAIK Little Tiles doesn't have a slide feature. I don't know if any of the "drawbridge" mods support chiseled blocks. But it's been 2 years with no joy, maybe can try asking the LT dev for sliding doors.
